runtimes/rust/src/lib.rs

```diff
@@ -248,6 +243,12 @@
 //! # }
 //! ```
 
+pub use exonum::runtime::{
```

Comment:
Nit / subjective: I wonder whether gathering these types in a module would make more sense:

```rust
pub use exonum::runtime as generic; // name TBD
```

This would be more structured, but more verbose when specifying imports.